.github/workflows/devcontainer.yaml

```diff
@@ -0,0 +1,64 @@
+name: devcontainer
+
+on:
+  push:
+    paths:
+      - .devcontainer/base-image/Dockerfile
+      - Taskfile.cargo-tools.yml
+  pull_request:
+    paths:
+      - .devcontainer/base-image/Dockerfile
+      - Taskfile.cargo-tools.yml
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}
+  cancel-in-progress: true
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: docker/metadata-action@v4
+        id: meta
+        with:
+          images: ghcr.io/${{ github.repository_owner }}/prql-devcontainer-base
+          tags: |
+            type=raw,latest
+
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v2
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - id: set-platforms
+        run: |
+          if [ "${{ github.ref }}" = "refs/heads/main" ]; then
+            echo "platforms=linux/amd64,linux/arm64" >>$GITHUB_OUTPUT
+            echo "push=true" >>$GITHUB_OUTPUT
+          else
+            echo "platforms=linux/amd64" >>$GITHUB_OUTPUT
+            echo "push=false" >>$GITHUB_OUTPUT
+          fi
```

Comment:
@eitsupi On reflection it's probably simpler to run both architectures on tests, not just amd64. Again — we should be liberal with workflows...

Reply:
Ok, I will fix this.
There is a disadvantage that multi-platform builds can't load images, so we can't run tests on containers, etc., but since we are only doing builds at this point, it shouldn't be a problem.